jina/drivers/evaluate.py

```diff
@@ -132,12 +128,19 @@ def __init__(self,
 
     @property
     def single_field(self):
+        """Get single field."""
```

Comment:
Can u add here the comment. Property to guarantee compatibility when only one field is provided either as a string or as a unit length tuple

Reply:
Sure! Thanks!